ui/components/component-library/avatar-account/avatar-account.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import Jazzicon from '../../ui/jazzicon/jazzicon.component';
+import BlockieIdenticon from '../../ui/identicon/blockieIdenticon/blockieIdenticon.component';
+import { BaseAvatar } from '../base-avatar';
+
+import { SIZES } from '../../../helpers/constants/design-system';
+
+const Diameters = {
+  xs: '16',
+  sm: '24',
+  md: '32',
+  lg: '40',
+  xl: '48',
+};
+
+export const types = ['Jazzicon', 'Blockie'];
+
+const getStyles = (diameter) => ({
+  width: diameter,
+  height: diameter,
+  borderRadius: diameter / 2,
+  display: 'flex',
+});
```

Comment:
Could we reduce the javascript logic here and just use a classname and CSS? It doesn't seem like width, height or borderRadius is needed either?

Add `avatar-account__jazzicon` to the `<Jazzicon className="avatar-account__jazzicon" />` component and in `avatar-account.scss` add the scss

```
.avatar-account {
  &__jazzicon {
    display: flex; // fixes size issue
  }
}
```